common/supplemental/supplementalMetadata.xml

```diff
@@ -1572,11 +1585,8 @@ For terms of use, see http://www.unicode.org/copyright.html
 						
 			
             <!-- variants -->
-            <variantAlias type="AALAND" replacement="AX" reason="deprecated"/>
-            <variantAlias type="POLYTONI" replacement="POLYTON" reason="deprecated"/>
-            <variantAlias type="HEPLOC" replacement="ALALC97" reason="deprecated"/>
-			<variantAlias type="arevela" replacement="hy" reason="deprecated"/> <!-- arevela -->
-			<variantAlias type="arevmda" replacement="hyw" reason="deprecated"/> <!-- arevmda -->
+            <variantAlias type="aaland" replacement="AX" reason="deprecated"/>
```

Comment:
Should we keep the VARIANT in this file always in UPPER case? since we already use lower case for language code, UPPER case for territory code, and Capital case for Script code in this file already.